src/store/rewards/saga.ts

```diff
@@ -116,6 +122,22 @@ export function* closeRewardsTooltip() {
   }
 }
 
+export function* transferMeow(action) {
+  const { senderUserId, recipientUserId, amount } = action.payload;
+
+  try {
+    const result = yield call(transferMeowAPI, senderUserId, recipientUserId, amount);
+
+    if (result.success) {
+      yield put(setMeow(result.response.senderBalance));
```

Comment:
If we set Meow, this should handle live updates of the users total rewards. Then when rewards are fetched again, i.e. on reload, the same amount of meow should be displayed based on the db updates when hitting transferMeowApi